docs/developers/documentation/index.md

```diff
@@ -1,79 +1,105 @@
 (docs_contributing_guide)=
 # Contributing Documentation
 
-This guide will teach you how to submit new documents to napari's usage documentation.
+This guide will teach you how to submit new documents to napari's usage
+documentation.
+
+## Organization of the documentation
+
+The napari documentation is built from sources located at the
+[napari/docs](https://github.com/napari/docs) repository on GitHub. That
+repository is where all the narrative documentation (e.g. tutorials, how-to
+guides) pull requests should be made. Meanwhile, changes to docstrings or to the
+[examples gallery](https://napari.org/gallery) should be made to the
+[napari/napari](https://github.com/napari/napari) repository.
 
 ## Prerequisites
 
-Prerequisites depend on the type of contribution you wish to make. In general
+Prerequisites depend on the type of contribution you wish to make. In general,
 you will require:
 
-- Some familiarity with [`git`](https://git-scm.com)
-- A [GitHub](https://github.com) account
+- Some familiarity with [`git`](https://git-scm.com);
+- A [GitHub](https://github.com) account.
+
+You should first [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+and clone the `napari/docs` repo to your machine, and navigate to the resulting
+folder:
+
+```bash
+git clone git@github.com:<your-username>/docs.git napari-docs
+cd napari-docs/
+```
 
-If you wish to add/amend documentation that does not contain code, you
-will also require:
+- **If you wish to add/amend documentation that does not contain code, you will
+  require a clean conda environment with napari docs dependencies installed.**
```

Comment:
Is this strictly true? I was thinking that with the docs being in a separate repo people could use GitHub.dev or the normal "pencil" tool on GitHub to edit docs, because they are all .md files.
If true, this would massively lower the "activation energy required".

Reply:
Fair point, in the docs meeting (https://hackmd.io/10DKrIMhREmk9sa_QRirlg) we did talk about wanting to highlight the “upload via web interface” workflow - if you are editing or adding just one file, this can be done via the github web interface then just use the CI to view the built documentation.

I think you can quite easily upload several files via web interface: https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository
However it is not as easy to edit multiple files and open one PR: https://stackoverflow.com/questions/17815895/can-i-edit-two-files-then-make-one-commit-using-github-web-based-editor - I am not sure this is necessarily easier than doing it via git...

I think we could update the docs to say that if you are adding file(s) or just amending one file, this can be done via web interface without installing git or making a new environment. This may be outside the scope of this PR though and could be done after this is merged.

Reply:
Fair! I'll open an issue for this.

Reply:
Here you go: #34